bin/oay/oay.toml.example

```diff
@@ -0,0 +1,9 @@
+[backend]
+type = "s3"
+
+access_key_id = "access_key_id"
+secret_access_key = "secret_access_key"
```

Comment:
May you add some comments about how to get available parameters?

Reply:
Later on.